js/login.js

```diff
@@ -0,0 +1,68 @@
+import { browserHistory } from 'react-router';
+import Service from './service.js';
+import UserData from './user-data.js';
+
+export default {
+  clearLoggedInParam(locationObj) {
+    let query = locationObj.query;
+
+    delete query.loggedin;
+
+    browserHistory.replace({
+      pathname: locationObj.pathname,
+      query: query
+    });
+  },
+  verifyLoggedInStatus(currentLocation, callback) {
```

Comment:
can we rename `callback` to have a name that actually indicates when it's getting called back, like `onUserStatusFinalised` or something? This makes understanding how the code work without first reading the entire file much easier. Or, given that we either call it with nothing or an error, `finishUserUpdate`